<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Røund Up</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(206, 211, 196);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>-10</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; color:#9791ae;&quot;&gt;Røund Up&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>181</width>
      <height>181</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>206</red>
          <green>211</green>
          <blue>196</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>212</red>
          <green>203</green>
          <blue>229</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Dark">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>124</red>
          <green>127</green>
          <blue>101</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Mid">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>190</red>
          <green>168</green>
          <blue>170</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>28</red>
          <green>29</green>
          <blue>33</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>206</red>
          <green>211</green>
          <blue>196</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>206</red>
          <green>211</green>
          <blue>196</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>140</red>
          <green>160</green>
          <blue>215</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Link">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>140</red>
          <green>160</green>
          <blue>215</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="LinkVisited">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>161</red>
          <green>77</green>
          <blue>160</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>206</red>
          <green>211</green>
          <blue>196</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>212</red>
          <green>203</green>
          <blue>229</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Dark">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>124</red>
          <green>127</green>
          <blue>101</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Mid">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>190</red>
          <green>168</green>
          <blue>170</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>28</red>
          <green>29</green>
          <blue>33</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>206</red>
          <green>211</green>
          <blue>196</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>206</red>
          <green>211</green>
          <blue>196</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>140</red>
          <green>160</green>
          <blue>215</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Link">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>140</red>
          <green>160</green>
          <blue>215</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="LinkVisited">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>161</red>
          <green>77</green>
          <blue>160</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>124</red>
          <green>127</green>
          <blue>101</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>206</red>
          <green>211</green>
          <blue>196</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>212</red>
          <green>203</green>
          <blue>229</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Dark">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>124</red>
          <green>127</green>
          <blue>101</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Mid">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>190</red>
          <green>168</green>
          <blue>170</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>124</red>
          <green>127</green>
          <blue>101</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>124</red>
          <green>127</green>
          <blue>101</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>206</red>
          <green>211</green>
          <blue>196</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>206</red>
          <green>211</green>
          <blue>196</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Link">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>140</red>
          <green>160</green>
          <blue>215</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="LinkVisited">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>161</red>
          <green>77</green>
          <blue>160</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../../../resources/icon_detailed.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>261</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
selection-background-color: rgb(151, 145, 174);</string>
   </property>
   <widget class="QMenu" name="menuOptionen">
    <property name="title">
     <string>Optionen</string>
    </property>
    <addaction name="actionBeenden"/>
    <addaction name="separator"/>
    <addaction name="actionHilfe"/>
   </widget>
   <widget class="QMenu" name="menuKlausur">
    <property name="title">
     <string>Klausur</string>
    </property>
    <addaction name="actionExamView"/>
    <addaction name="actionExamAdd"/>
    <addaction name="actionExamRemove"/>
   </widget>
   <widget class="QMenu" name="menuKurse">
    <property name="title">
     <string>Kurse</string>
    </property>
    <addaction name="actionCourseView"/>
    <addaction name="actionCourseAdd"/>
    <addaction name="actionCourseRemove"/>
   </widget>
   <widget class="QMenu" name="menuErgebnisse">
    <property name="title">
     <string>Ergebnisse</string>
    </property>
    <addaction name="actionResultView"/>
    <addaction name="actionResultAdd"/>
    <addaction name="actionResultRemove"/>
   </widget>
   <addaction name="menuOptionen"/>
   <addaction name="menuKlausur"/>
   <addaction name="menuKurse"/>
   <addaction name="menuErgebnisse"/>
  </widget>
  <action name="actionExamView">
   <property name="text">
    <string>Ansehen</string>
   </property>
  </action>
  <action name="actionResultView">
   <property name="text">
    <string>Ansehen</string>
   </property>
  </action>
  <action name="actionCourseView">
   <property name="text">
    <string>Ansehen</string>
   </property>
  </action>
  <action name="actionExamAdd">
   <property name="text">
    <string>Hinzufügen</string>
   </property>
  </action>
  <action name="actionBeenden">
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
  <action name="actionCourseAdd">
   <property name="text">
    <string>Hinzufügen</string>
   </property>
  </action>
  <action name="actionResultAdd">
   <property name="text">
    <string>Hinzufügen</string>
   </property>
  </action>
  <action name="actionExamRemove">
   <property name="text">
    <string>Entfernen</string>
   </property>
  </action>
  <action name="actionCourseRemove">
   <property name="text">
    <string>Entfernen</string>
   </property>
  </action>
  <action name="actionResultRemove">
   <property name="text">
    <string>Entfernen</string>
   </property>
  </action>
  <action name="actionHilfe">
   <property name="text">
    <string>Hilfe</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionBeenden</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
